modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/data_set/filters/modals/selection_filter/source_type/Headless.tsx

```diff
@@ -49,7 +49,7 @@ interface IApiRestApplicationModalContentProps {
 	restSchemaValidationError: boolean;
 }
 
-function ApiRestApplication({
+function Headless({
```

Comment:
I renamed `ApiRestApplication` to `Headless`, to rhyme with headless type of actions. `ApiRestApplication` sounds just weird for a source.